examples/customer-satisfaction-survey-analysis/.github/README.md

```diff
@@ -0,0 +1 @@
+../Zendesk customer satisfaction survey analysis.md
```

Comment:
Can we fix the symlink here?

Reply:
check for other places

Reply:
All the symlinks are already working (e.g. https://github.com/NipunaRanasinghe/module-ballerinax-zendesk/tree/docs/examples/customer-satisfaction-survey-analysis)